Grade: A1; To perform analysis on a single cell dataset and subsequent interpretation of the data and results in Python and R.
Please note, in the jupyter notebook the implementation is given in Python and the R equivalent code is mentioned as commented blocks.

Input Required:
1. Standard 10x-Genomics-formatted mtx directory/ directories.

What the code does:
1. Inputs the 10X Chromium dataset of a γδ T cell dataset with three wild-type (WT) samples and one knock-out (KO) using scanpy in Python and Seurat in R.
2. The code implements an analysis pipeline in Python and performs quality control of the samples.
3. It integrates the data with different methods and  annotates the different clusters.
4. Finally it performs differential expression analysis with two tools and builds a trajectory.
